src/cli/mod.rs

```diff
@@ -50,6 +50,8 @@ enum Command {
         /// Connection string for the Postgres
         #[clap(env = "PG_CON", default_value = "")]
         db_con: String,
+        #[clap(env = "SIGNER_ENTRYPTION_KEY", default_value = "")]
```

Comment:
we need to think about the default - if daemon is started with `dev` mode it needs to be set to something.